wandb/sdk/lib/debug_log.py

```diff
@@ -69,6 +69,22 @@ def _log(
         logger.info(line)
 
 
+def _record_msg_type(record: "pb.Record") -> str:
+    msg_type = str(record.WhichOneof("record_type"))
+    if msg_type == "request":
+        request = record.request
+        msg_type = str(request.WhichOneof("request_type"))
+    return msg_type
+
+
+def _result_msg_type(result: "pb.Result") -> str:
+    msg_type = str(result.WhichOneof("result_type"))
+    if msg_type == "response":
+        response = result.response
+        msg_type = str(response.WhichOneof("response_type"))
+    return msg_type
```

Comment:
do you think these functions would be useful outside of this file?
i.e does it make sense to move them to `proto_util`?